components/brave_perf_predictor/browser/third_parties.h

```diff
@@ -0,0 +1,4363 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_PERF_PREDICTOR_BROWSER_THIRD_PARTIES_H_
+#define BRAVE_COMPONENTS_BRAVE_PERF_PREDICTOR_BROWSER_THIRD_PARTIES_H_
+
+#include <string>
+
+namespace brave_perf_predictor {
+
+const char static_third_party_config[] = R"--(
```

Comment:
this needs a follow-up to convert it to a resource in a grd file - see bat_ads_resources.grd for an example